### PR TITLE
refactor: use useForm hook in Linear and Zoom extensions

### PR DESCRIPTION
**Pull request by thomaslombart**
## Description

Refactor the Linear and Zoom form commands to use the new `useForm` hook.

**Pending for a function to reset the values before opening the PR.**

## Screencast

No screencast as no functionality has changed.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/zoom/src/components/CreateMeetingForm.tsx [hunk 1]
```diff
@@ -17,103 +17,92 @@ type MeetingFormProps = {
...
+      } catch (error) {
+        toast.style = Toast.Style.Failure;
+        toast.title = "Failed to schedule meeting";
+        toast.message = getErrorMessage(error);
+      }
+    },
+    validation: {
+      start_time: FormValidation.Required,
+      duration: (value) => {
+        if (isNaN(parseInt(value ?? ""))) {
```
Comment (mathieudutour) [inline extensions/zoom/src/components/CreateMeetingForm.tsx:75]: this isn't required, right?
```suggestion
        if (value && isNaN(parseInt(value ?? ""))) {
```

Comment (thomaslombart) [reply extensions/zoom/src/components/CreateMeetingForm.tsx:75]: Indeed, the duration isn't required.

Comment (mathieudutour) [reply extensions/zoom/src/components/CreateMeetingForm.tsx:75]: right so the check won't work because `isNaN(parseInt('')) === true`

Comment (thomaslombart) [reply extensions/zoom/src/components/CreateMeetingForm.tsx]: Oh, yes indeed. Thanks for pointing it out.

**Review by mathieudutour — APPROVED** [review]

**Comment by thomaspaulmann** [comment]
linear 2022-09-22 at 15 37 08

I get an error without setting a due date 😥

**Pull request merged**